turn_alerts/views.py

```diff
@@ -0,0 +1,137 @@
+import re
+import time
+
+from prometheus_client import Counter, Histogram
+from rest_framework import generics, permissions, status
+from rest_framework.response import Response
+
+from turn_alerts.serializers import (
+    ContactsPayloadSerializer,
+    Status_ErrorPayloadSerializer,
+    StatusPayloadSerializer,
+    VendorPayloadSerializer,
+)
+
+from .models import TurnAlerts
+from .tasks import start_turn_journey
+
+request_count = Counter(
+    "turn_alerts_request_total", "Total number of requests to TurnAlertsLayerView"
+)
+processing_time = Histogram(
+    "turn_alerts_request_processing_seconds", "Processing time of requests in seconds"
+)
```

Comment:
I think we're more interested in how many messages and events are coming through.

Messages grouped by fallback, direction, type
Events grouped by status, code etc

I think in the gist we group events by type as an example